ToolTestingUtils/ToolTestingScript.ps1

```diff
@@ -0,0 +1,41 @@
+param (    
+    [parameter(mandatory=$true, HelpMessage="Azure Stack One Node host address or name such as '1.2.3.4'")]
+    [string] $HostComputer,
+    [parameter(HelpMessage="Domain FQDN of this Azure Stack Instance")]
+    [string] $Domain = "azurestack.local",
+    [parameter(HelpMessage="NAT computer name in this Azure Stack Instance")]
+    [string] $natServer = "MAS-BGPNAT01",
+    [parameter(HelpMessage="Administrator user name of this Azure Stack Instance")]
+    [string] $AdminUser = "administrator",
+    [parameter(mandatory=$true, HelpMessage="Administrator password used to deploy this Azure Stack instance")]
+    [securestring] $AdminPassword,
+    [parameter(mandatory=$true, HelpMessage="The AAD service admin user name of this Azure Stack Instance")]
+    [string] $AadServiceAdmin,
```

Comment:
no changes immediately, we should call $AadServiceAdmin as simply ServiceAdmin considering the ADFS support in future